src/routes/title-edit.js

```diff
@@ -0,0 +1,93 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import { createResolver } from '../redux';
+import Title from '../redux/title';
+import Resource from '../redux/resource';
+
+import View from '../components/title/edit';
+
+class TitleShowRoute extends Component {
+  static propTypes = {
+    match: PropTypes.shape({
+      params: PropTypes.shape({
+        titleId: PropTypes.string.isRequired
+      }).isRequired
+    }).isRequired,
+    model: PropTypes.object.isRequired,
+    getTitle: PropTypes.func.isRequired,
+    updateResource: PropTypes.func.isRequired,
+    updateRequest: PropTypes.object
+  };
+
+  static contextTypes = {
+    router: PropTypes.shape({
+      history: PropTypes.shape({
+        push: PropTypes.func.isRequired
+      }).isRequired
+    }).isRequired
+  };
+
+  componentWillMount() {
+    let { match, getTitle } = this.props;
+    let { titleId } = match.params;
+    getTitle(titleId);
+  }
+
+  componentWillReceiveProps(nextProps) {
+    let { match, getTitle } = nextProps;
+    let { titleId } = match.params;
+
+    if (titleId !== this.props.match.params.titleId) {
+      getTitle(titleId);
+    }
+  }
+
+  componentDidUpdate(prevProps) {
+    if (!prevProps.updateRequest.isResolved && this.props.updateRequest.isResolved) {
+      this.context.router.history.push(
+        `/eholdings/titles/${this.props.model.id}`,
+        { eholdings: true }
+      );
+    }
+  }
+
+  titleEditSubmitted = (values) => {
```

Comment:
Nice on making edit against first package in the front end at least temporarily until available in mod-kb-ebsco